debian/control

```diff
@@ -95,7 +96,8 @@ Conflicts: dde-file-manager-preview,
  dde-file-manager-preview-plugins,
  dde-file-manager-plugins,
  dde-file-manager-daemon-plugins,
- dde-file-manager-common-plugins
+ dde-file-manager-common-plugins,
+ libheif1
```

Comment:
It looks like there is no need to add libheif1, check the dependencies as libheif-dev already depends on libheif1